lib/img-workers.js

```diff
@@ -30,13 +30,17 @@ exports.upload = function upload(id, payload, headers) {
     };
     logger.verbose('upload', url);
     payload.pipe(request.post(url, opts, function(err, res, body) {
+      var nestedError = null;
+      if (body && body[0] && body[0].error) {
```

Comment:
from my logs this returned an array with an error  and not `body.error`. I wonder if it ever returns body.error?